### PR TITLE
Feature/design styles

### PR DESCRIPTION
**Pull request by Ritatrcr**

### CODE REVIEW

File: src/pages/home.jsx [hunk 1]
```diff
@@ -1,9 +1,30 @@
-import React from 'react'
+import React from 'react';
+import Card from '../components/designSystem/card';
+import Button from '../components/designSystem/button';
+
 
 const Home = () => {
+  // Función que maneja el clic del botón
```
Comment (juancho20sp) [inline src/pages/home.jsx:8]: Document in english please

File: src/pages/home.jsx [hunk 2]
```diff
@@ -1,9 +1,30 @@
...
-    <div>Home</div>
-  )
-}
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        height: '100vh',
+        backgroundColor: '#1F1F1F',
```
Comment (juancho20sp) [inline src/pages/home.jsx:20]: Can we use tailwind here please?

File: src/pages/home.jsx [hunk 3]
```diff
@@ -1,9 +1,30 @@
...
+      }}
+    >
+      <Card>
+        <p style={{ color: '#FFFFFF' }}>Aquí puedes chatear con nuestro equipo</p>
```
Comment (juancho20sp) [inline src/pages/home.jsx:24]: Can we use tailwind here please?

File: src/components/designStyles/card.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    color: '#000', 
+    padding: '10px 30px',
+    fontSize: '12px',
+    fontWeight: 'bold',
+    borderRadius: '5px',
+    border: 'none',
+    cursor: 'pointer',
+    display: 'inline-block',
+    textAlign: 'center',
+  };
```
Comment (juancho20sp) [inline src/components/designStyles/card.jsx]: Can we use tailwind here please?

**Review by JuanGomePer — APPROVED** [review]

File: src/components/designStyles/button.jsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    color: '#000', 
+    padding: '10px 30px',
+    fontSize: '12px',
+    fontWeight: 'bold',
+    borderRadius: '5px',
+    border: 'none',
+    cursor: 'pointer',
+    display: 'inline-block',
+    textAlign: 'center',
+  };
```
Comment (juancho20sp) [inline src/components/designStyles/button.jsx]: Can we use tailwind here please?

**Pull request merged**